sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/HL7v2IO.java

```diff
@@ -509,13 +561,17 @@ public ListHL7v2Messages withInitialSplitDuration(Duration initialSplitDuration)
      * @param filter the filter
      */
     ListHL7v2MessagesFn(String filter) {
-      this(StaticValueProvider.of(filter), null);
+      this(StaticValueProvider.of(filter), null, null);
```

Comment:
We pass in null to the constructor and don't handle that case in the switch statement. How about passing in the default?